src/memory-analyzer/analyze_symbol.h

```diff
@@ -91,6 +91,70 @@ class gdb_value_extractort
   ///   value of `symbol`.
   std::map<memory_addresst, exprt> values;
 
+  struct memory_scopet
+  {
+    memory_addresst begin;
+    mp_integer byte_size;
+    irep_idt name;
+
+    memory_scopet() = delete;
```

Comment:
I'm not sure that's necessary if you explicitly define some other constructor?

Reply:
You are right. Removed.